src/proxy-ui-api/frontend/src/views/KeyDetails/KeyDetails.vue

```diff
@@ -295,7 +295,7 @@ export default Vue.extend({
         .finally(() => (this.deleting = false));
     },
     setKeyName(): void {
-      if (this.key.name === '') {
+      if (this.key.name === undefined || this.key.name === '') {
```

Comment:
This does not actually check for `null`s but instead `undefined`s. Still it does work in practice. 

There is also helpers.ts#isEmpty that could be used for this kind of check.

Reply:
Changed this to `helpers.ts#isEmpty`.